src/utils/rari.js

```diff
@@ -837,12 +837,35 @@ export const getWithdrawalFeeRate = (rariPool: RariPool) => {
     });
 };
 
-export const getRariClaimRgtTransaction = (senderAddress: string, amount: number, txFeeInWei?: BigNumber) => {
+export const getClaimRtgFee = async () => {
+  const distributionStartBlock = 11094200;
+  const distributionPeriod = 390000;
+  const distributionEndBlock = distributionStartBlock + distributionPeriod;
+  const initialClaimFee = 0.33;
```

Comment:
Just wondering, do any of these need to go into our envConfig / remote config?

```
  const distributionStartBlock = 11094200;
  const distributionPeriod = 390000;;
  const initialClaimFee = 0.33;
```

Reply:
No, because they're not going to change. Unless there is another liquidity mining program :D

Reply:
Haha, fair! Thanks for clearing that up.